packages/vpnc.rb

```diff
@@ -0,0 +1,28 @@
+require 'package'
+
+class Vpnc < Package
+  description 'client for cisco vpn concentrator'
+  homepage 'https://www.unix-ag.uni-kl.de/~massar/vpnc/'
+  version '0.5.3'
+  source_url 'https://www.unix-ag.uni-kl.de/~massar/vpnc/vpnc-0.5.3.tar.gz'
+  source_sha256 '46cea3bd02f207c62c7c6f2f22133382602baeda1dc320747809e94881414884'
+
+  depends_on 'libgcrypt'
+  depends_on 'openssl'
+  depends_on 'perl'
+
+  def self.build
+    system "sed -i 's,\./makeman.pl,perl makeman.pl,' Makefile"
+    system "sed -i 's,ETCDIR=/etc/vpnc,ETCDIR=#{CREW_PREFIX}/etc/vpnc,' Makefile"
+    system "make"
+  end
+
+  def self.install
+    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "mkdir -p #{CREW_DEST_DIR}#{CREW_PREFIX}/etc/vpnc"
+    FileUtils.cd("#{CREW_DEST_DIR}#{CREW_PREFIX}/etc/vpnc") do
+      system "wget http://git.infradead.org/users/dwmw2/vpnc-scripts.git/blob_plain/HEAD:/vpnc-script"
```

Comment:
Huh?  You install vpnc-script by 'make ... install' three lines above.  Then, you download another vpnc-script at this line.  Didn't you check the output from `crew install vpnc` when you create it?  It appears like...
```
$ crew install vpnc
...
perl makeman.pl
Preconfiguring package...
install -d /usr/local/tmp/crew/dest/usr/local/etc/vpnc /usr/local/tmp/crew/dest/usr/local/bin /usr/local/tmp/crew/dest/usr/local/sbin /usr/local/tmp/crew/dest/usr/local/share/man/man1 /usr/local/tmp/crew/dest/usr/local/share/man/man8 /usr/local/tmp/crew/dest/usr/local/share/doc/vpnc
if [ "`uname -s | cut -c-6`" = "CYGWIN" ]; then \
        install vpnc-script-win /usr/local/tmp/crew/dest/usr/local/etc/vpnc/vpnc-script; \
        install vpnc-script-win.js /usr/local/tmp/crew/dest/usr/local/etc/vpnc; \
else \
        install vpnc-script /usr/local/tmp/crew/dest/usr/local/etc/vpnc; \
fi
install -m600 vpnc.conf /usr/local/tmp/crew/dest/usr/local/etc/vpnc/default.conf
install -m755 vpnc-disconnect /usr/local/tmp/crew/dest/usr/local/sbin
install -m755 pcf2vpnc /usr/local/tmp/crew/dest/usr/local/bin
install -m644 vpnc.8 /usr/local/tmp/crew/dest/usr/local/share/man/man8
install -m644 pcf2vpnc.1 /usr/local/tmp/crew/dest/usr/local/share/man/man1
install -m644 cisco-decrypt.1 /usr/local/tmp/crew/dest/usr/local/share/man/man1
install -m644 COPYING /usr/local/tmp/crew/dest/usr/local/share/doc/vpnc
install -m755 vpnc /usr/local/tmp/crew/dest/usr/local/sbin
install -m755 cisco-decrypt /usr/local/tmp/crew/dest/usr/local/bin
--2017-08-09 08:12:21--  http://git.infradead.org/users/dwmw2/vpnc-scripts.git/blob_plain/HEAD:/vpnc-script
Resolving git.infradead.org... 65.50.211.133
Connecting to git.infradead.org|65.50.211.133|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [text/plain]
Saving to: 'vpnc-script.1'

vpnc-script.1                         [ <=>                                                        ]  25.46K  --.-KB/s    in 0.1s    

2017-08-09 08:12:22 (213 KB/s) - 'vpnc-script.1' saved [26067]

Checking...
```